.github/release_notes_template/template.hbs

```diff
@@ -38,10 +38,10 @@ Bug Fixes:
     {{/ifCommitType}}
   {{/commits}}
 {{/ifContainsType}}
-{{! Deprecated APIs section }}
```

Comment:
and setting as "Deprecated APIs/components section"?

Reply:
Fixed. Please note that this is just a comment though. Do you think it would be better to have it as the title of the section? Currently it's simply "Depreated"...

Reply:
Yes, I was thinking on both